pkg/storage/sqlstorage/driver.go

```diff
@@ -143,41 +144,39 @@ func (d *Driver) GetConfiguration(ctx context.Context, key string) (string, erro
 	return value, nil
 }
 
-func (d *Driver) Initialize(ctx context.Context) (err error) {
+func (d *Driver) Initialize(ctx context.Context) error {
 	sharedlogging.GetLogger(ctx).Debugf("Initialize driver %s", d.name)
 
-	if err = d.db.Initialize(ctx); err != nil {
-		return
+	if err := d.db.Initialize(ctx); err != nil {
+		return fmt.Errorf("db.Initialize: %w", err)
 	}
 
+	var err error
 	d.systemSchema, err = d.db.Schema(ctx, SystemSchema)
 	if err != nil {
-		return
+		return fmt.Errorf("db.Schema: %w", err)
 	}
 
-	if err = d.systemSchema.Initialize(ctx); err != nil {
-		return
+	if err := d.systemSchema.Initialize(ctx); err != nil {
+		return fmt.Errorf("systemSchema.Initialize: %w", err)
 	}
 
 	q, args := sqlbuilder.
 		CreateTable(d.systemSchema.Table("ledgers")).
 		Define("ledger varchar(255) primary key, addedAt timestamp").
 		IfNotExists().
 		BuildWithFlavor(d.systemSchema.Flavor())
-
-	_, err = d.systemSchema.ExecContext(ctx, q, args...)
-	if err != nil {
-		return err
+	if _, err := d.systemSchema.ExecContext(ctx, q, args...); err != nil {
```

Comment:
You don't have to force to inline "if err" clauses everywhere (not a blocker, but it modify code which don't have to be modified, just for personal preference).